src/ClientInstance.cpp

```diff
@@ -686,7 +686,7 @@ PVR_ERROR CVNSIClientInstance::GetAvailableRecordings(kodi::addon::PVRRecordings
     tag.SetPlotOutline(tag.GetEpisodeName());
     tag.SetPlot(vresp->extract_String());
     tag.SetDirectory(vresp->extract_String());
-    tag.SetRecordingId(StringUtils::Format("%i", vresp->extract_U32()));
+    tag.SetRecordingId(kodi::tools::StringUtils::Format("%i", vresp->extract_U32()));
```

Comment:
Thing for here a `std::to_string(vresp->extract_U32())` can be enough and `StringUtils` no more needed.

Reply:
Cool